udata/core/spatial/api.py

```diff
@@ -32,6 +35,35 @@
     'size', type=int, help='The amount of suggestion to fetch',
     location='args', default=10)
 
+dataset_parser = api.parser()
+dataset_parser.add_argument(
+    'with_dynamic', type=bool, help='Append dynamic datasets',
```

Comment:
Is it possible to reduce to a single word ? `dynamic=true` seems sufficient and explicit